src/vs/code/electron-main/app.ts

```diff
@@ -265,7 +266,8 @@ export class CodeApplication extends Disposable {
 					cli: this.environmentMainService.args,
 					urisToOpen: macOpenFileURIs,
 					gotoLineMode: false,
-					preferNewWindow: true /* dropping on the dock or opening from finder prefers to open in a new window */
+					preferNewWindow: true, /* dropping on the dock or opening from finder prefers to open in a new window */
+					remoteAuthority: undefined /* always local */
```

Comment:
Is that needed here though? Shouldn't the fact that we have `urisToOpen` be enough to decide on the `remoteAuthority`?